src/pages/benefits/benefitFormUI/BenefitFormUI.tsx

```
@@ -63,27 +72,50 @@ const BenefitFormUI: React.FC = () => {
     const getSchemaData = async () => {
       if (id) {
         const result = await getSchema(id);
-        const targetTag =
-          result?.responses?.[0]?.message?.order?.items?.[0]?.tags?.find(
-            (tag: any) => tag?.descriptor?.code === "benefit_schema"
+        const schemaTag =
+          result?.responses?.[0]?.message?.catalog?.providers?.[0]?.items?.[0]?.tags?.find(
+            (tag: any) => tag?.descriptor?.code === "applicationForm"
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)